internal/observability/graph.go

```diff
@@ -477,6 +478,7 @@ func newEdge(logger *zap.Logger, meter *metrics.Scope, metricTagsBlocklist []str
 		zap.String("encoding", string(req.Encoding)),
 		zap.String("routingKey", req.RoutingKey),
 		zap.String("routingDelegate", req.RoutingDelegate),
+		zap.String("sourceProcedure", req.CallerProcedure),
```

Comment:
Why doesn't the log key match the property name?

When the `transport.Request` object is logged, the `CallerProcedure` property is logged as `callerProcedure`.
https://github.com/yarpc/yarpc-go/blob/dev/api/transport/request.go#L107

Reply:
Also the on-the-wire representation is `rpc-caller-procedure` , not `source-procedure`, so I think `callerProcedure` makes more sense in the logs.

https://github.com/yarpc/yarpc-go/blob/9d0cab62f323aac0e60616fd3952d5b36ca85479/transport/grpc/headers.go#L40
https://github.com/yarpc/yarpc-go/blob/f83a481b968915f1d834d2e66daf799ccd521361/transport/tchannel/header.go
https://github.com/yarpc/yarpc-go/blob/f83a481b968915f1d834d2e66daf799ccd521361/transport/http/constants.go#L45